recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - gdal 2.1.*
     - ghostscript
     - libnetcdf 4.4.*
```

Comment:
Yeah, I need to get ```libnetcdf``` to fix ```gdal``` too. Is there a way for me to re-trigger the CIs without amending the commit?